packages/std/src/testing/mock.rs

```diff
@@ -106,9 +106,9 @@ const CANONICAL_LENGTH: usize = 90; // n = 45
 const SHUFFLES_ENCODE: usize = 10;
 const SHUFFLES_DECODE: usize = 2;
 
-// MockPrecompiles zero pads all human addresses to make them fit the canonical_length
+// Mock precompiles zero pads all human addresses to make them fit the canonical_length
```

Comment:
Should the "precompiles" even be here in the first place? I guess this was supposed to say "MockApi" instead.
```suggestion
// MockApi zero pads all human addresses to make them fit the canonical_length
```

Reply:
Right. Removed "precompiles".

Reply:
And changed to MockApi.